src/driver/inceptionv3.cpp

```diff
@@ -27,1916 +27,2838 @@
+        x_main_module_580,
+        x_main_module_312,
+        x_main_module_311,
+        x_main_module_310,
+        x_main_module_309); // NOLINT(modernize-raw-string-literal)
+    auto x_main_module_582 = mmain->add_instruction(migraphx::make_op("relu"), x_main_module_581);
+    auto x_main_module_583 = mmain->add_instruction(
+        migraphx::make_op("convolution",
+                          migraphx::from_json_string(
+                              "{\"dilation\":[1,1],\"group\":1,\"padding\":[3,0,3,0],\"padding_"
+                              "mode\":0,\"stride\":[1,1],\"use_dynamic_same_auto_pad\":0}")),
+        x_main_module_582,
+        x_main_module_308); // NOLINT(modernize-raw-string-literal)
+    auto x_main_module_584 = mmain->add_instruction(
+        migraphx::make_op(
+            "batch_norm_inference",
+            migraphx::from_json_string("{\"bn_mode\":1,\"epsilon\":0.0010000000474974513,"
+                                       "\"momentum\":0.8999999761581421}")),
+        x_main_module_583,
+        x_main_module_307,
+        x_main_module_306,
+        x_main_module_305,
+        x_main_module_304); // NOLINT(modernize-raw-string-literal)
+    auto x_main_module_585 = mmain->add_instruction(migraphx::make_op("relu"), x_main_module_584);
     auto x_main_module_586 = mmain->add_instruction(
-        migraphx::make_op("broadcast", migraphx::from_json_string("{axis:1,out_lens:[1,384,8,8]}")),
-        x_main_module_69);
-    auto x_main_module_587 =
-        mmain->add_instruction(migraphx::make_op("add"), x_main_module_585, x_main_module_586);
+        migraphx::make_op("convolution",
+                          migraphx::from_json_string(
+                              "{\"dilation\":[1,1],\"group\":1,\"padding\":[0,0,0,0],\"padding_"
+                              "mode\":0,\"stride\":[1,1],\"use_dynamic_same_auto_pad\":0}")),
+        x_main_module_573,
+        x_main_module_303); // NOLINT(modernize-raw-string-literal)
+    auto x_main_module_587 = mmain->add_instruction(
+        migraphx::make_op(
+            "batch_norm_inference",
+            migraphx::from_json_string("{\"bn_mode\":1,\"epsilon\":0.0010000000474974513,"
+                                       "\"momentum\":0.8999999761581421}")),
+        x_main_module_586,
+        x_main_module_302,
+        x_main_module_301,
+        x_main_module_300,
+        x_main_module_299); // NOLINT(modernize-raw-string-literal)
     auto x_main_module_588 = mmain->add_instruction(migraphx::make_op("relu"), x_main_module_587);
     auto x_main_module_589 = mmain->add_instruction(
-        migraphx::make_op(
-            "convolution",
-            migraphx::from_json_string(
-                "{dilation:[1,1],group:1,padding:[0,1,0,1],padding_mode:0,stride:[1,1]}")),
+        migraphx::make_op("convolution",
+                          migraphx::from_json_string(
+                              "{\"dilation\":[1,1],\"group\":1,\"padding\":[3,0,3,0],\"padding_"
```

Comment:
Just a naive check/question on this. 

Is it supposed to be  
```
.....bunch of stuff\"padding_""mode\" .........}"
                             ^no escape is deliberate here?
```

Reply:
This is out of date with the new code change, but basically the compiler knows that this is a multi-line string, so it will just concatenate them together without inserting another quotation mark.

Reply:
Sounds good ok.